.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
 script:
   - source test/env.sh
   - ./rebar3 compile
-  - ./rebar3 do xref, ct, coveralls send
+  - sudo env PATH=$PATH ./rebar3 do xref, ct, coveralls send
```

Comment:
```sudo -E ...``` should do the trick an keep all environment variables

Reply:
I have tried this in the beginning - didn't work. Then I switched to `sudo env PATH=..`. But maybe `dist` is the reason. Will check with `dist: xenial`.

Reply:
the sudo will also undo the effect of the ```source test/env.sh``` script.